src/algebraic_geometry/stalks.lean

```diff
@@ -0,0 +1,63 @@
+import algebraic_geometry.presheafed_space
+import category_theory.instances.Top.open_nhds
+import category_theory.instances.Top.stalks
+import category_theory.limits.limits
+
+universes v u v' u'
+
+open category_theory
+open category_theory.instances
+open category_theory.limits
+open algebraic_geometry
+open topological_space
+
+variables {C : Type u} [𝒞 : category.{v+1} C] [has_colimits.{v} C]
+include 𝒞
+
+open category_theory.instances.Top.presheaf
+
+namespace algebraic_geometry.PresheafedSpace
+
+def stalk (X : PresheafedSpace.{v} C) (x : X) : C := X.𝒪.stalk x
+
+def stalk_map {X Y : PresheafedSpace.{v} C} (α : X ⟶ Y) (x : X) : Y.stalk (α x) ⟶ X.stalk x :=
+(stalk_functor C (α x)).map (α.c) ≫ X.𝒪.stalk_pushforward C α x
```

Comment:
Is there a reason why this definition and the following lemmas are not in `category_theory/instances/Top/stalks`?

Reply:
I'd wanted to keep `PresheafedSpace` out of the `instances/Top/` folder.